pandas/core/arrays/datetimelike.py

```diff
@@ -1640,6 +1640,17 @@ def floor(self, freq, ambiguous="raise", nonexistent="raise"):
     def ceil(self, freq, ambiguous="raise", nonexistent="raise"):
         return self._round(freq, RoundTo.PLUS_INFTY, ambiguous, nonexistent)
 
+    # --------------------------------------------------------------
+    # Reductions
+
+    def any(self, *, axis: Optional[int] = None, skipna: bool = True):
+        # GH#33478
```

Comment:
Is there a reason why this isn't `GH 38723`?

Reply:
typo, should be #33479, which is about the long-term behavior we want, #38723 is about ensuring that the interim behavior is internally-consistent.  just updated